modules/storage-api/README.md

```diff
@@ -53,4 +53,4 @@ String dataRegion = dataStorageView.dateRegion();
 ```
 
 
-To get the data storage engine, you need to use `org.apache.ignite.internal.storage.DataStorageManager.engine(org.apache.ignite.configuration.schemas.store.DataStorageConfiguration)`.
\ No newline at end of file
+To get the data storage engine, you need to use `org.apache.ignite.internal.storage.DataStorageManager.engine(org.apache.ignite.configuration.schemas.store.DataStorageConfiguration)`.
```

Comment:
Looks like this whole document is now obsolete, it still mentions data regions

Reply:
This document is an obsolete in general, tables are not the part of configurations for some time, also. So, to rewrite this document and to review another docs I created the separate issue already https://issues.apache.org/jira/browse/IGNITE-21563